sky/shell/platform/mac/platform_service_provider.cc

```diff
@@ -79,8 +80,10 @@ void PlatformServiceProvider::ConnectToService(
   }
 #endif
 
-  LOG(INFO) << "The platform service provider cannot find a service for '"
-            << service_name.data() << "'";
+  LOG(INFO) << "No bundled service provider for '" << service_name.data()
```

Comment:
Ok, the dynamic service loader that eventually gets the request already logs if it fails. Will remove this log.